### PR TITLE
Created .linspace() entry

### LINKED ISSUE #5224: [Term Entry] PyTorch Tensors .linspace()
### Reason/inspiration (optional)

We would like a new entry on the `.linspace()` term under PyTorch. The entry should go in a new file under `docs/content/pytorch/concepts/tensors/terms/linspace/linspace.md`.

The entry should include:

- An introduction to the concept
- A `Syntax` section that provides the syntax for the concept
- An `Example` section that provides an example demonstrating the concept in use
- A `Codebyte Example` section that provides a codebyte example demonstrating the concept in use

Please refer to the [term entry template](https://github.com/Codecademy/docs/blob/main/documentation/term-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md) and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [ ] Issue is assigned

### PR DESCRIPTION
**Pull request by arisdelacruz**
### Description

This is an attempt to create an entry for the .linspace() function for Pytorch

### Issue Solved

Closes #5224 

### Type of Change

- Adding a new entry

### Checklist

- [ / ] All writings are my own.
- [ / ] My entry follows the Codecademy Docs style guide.
- [ / ] My changes generate no new warnings.
- [ / ] I have performed a self-review of my own writing and code.
- [ / ] I have checked my entry and corrected any misspellings.
- [ / ] I have made corresponding changes to the documentation if needed.
- [ / ] I have confirmed my changes are not being pushed from my forked `main` branch.
- [ / ] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [ / ] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

File: content/pytorch/concepts/tensors/terms/linspace/linspace.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  - 'Machine Learning'
+Tags:
+  - 'AI'
+  - 'Data Types'
+  - 'Deep Learning'
+  - 'Functions'
+CatalogContent:
+  - 'intro-to-py-torch-and-neural-networks'
+  - 'py-torch-for-classification'
+---
```
Comment (Sriparno08) [inline content/pytorch/concepts/tensors/terms/linspace/linspace.md:15]: Let's change the description a bit for uniqueness:

```suggestion
---
Title: '.linspace()'
Description: 'Returns a one-dimensional tensor of the specified size, whose values are equally spaced from the given starting and ending values.'
Subjects:
  - 'Data Science'
  - 'Machine Learning'
Tags:
  - 'AI'
  - 'Data Types'
  - 'Deep Learning'
  - 'Functions'
CatalogContent:
  - 'intro-to-py-torch-and-neural-networks'
  - 'py-torch-for-classification'
---
```

File: content/pytorch/concepts/tensors/terms/linspace/linspace.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+The **`.linspace()`** function in Pytorch is used to create a one-dimensional tensor of steps equally spaced points between the start and end values. It takes three arguments: the start value, the end value, and the number of steps.
+
+The function returns a one-dimensional tensor of equally spaced points between the start and end values.
```
Comment (Sriparno08) [inline content/pytorch/concepts/tensors/terms/linspace/linspace.md:19]: Modifications:

- Let's remove the second line as it's already mentioned in `Syntax`
- Let's remove the third line as it resembles the first line
- Let's change the first line a bit for uniqueness

```suggestion
The **`.linspace()`** function in PyTorch is used to return a one-dimensional tensor of the specified size, whose values are equally spaced from the given starting and ending values.
```

File: content/pytorch/concepts/tensors/terms/linspace/linspace.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+## Syntax
+
+```pseudo
+torch.linspace(start, end, steps)
+```
+
+- `start` is the starting value of the sequence.
+- `end` is the ending value of the sequence.
+- `steps` is the number of steps to be taken between the start and end values.
```
Comment (Sriparno08) [inline content/pytorch/concepts/tensors/terms/linspace/linspace.md:29]: Let's do some minor changes in the bulleted list:

```suggestion
- `start`: The starting value to be used.
- `end`: The ending value to be used.
- `steps`: The number of steps to be taken between the starting and ending values.
```

File: content/pytorch/concepts/tensors/terms/linspace/linspace.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+This function is particularly useful when you need to create a tensor of equally spaced points for plotting graphs or for other numerical computations.
```
Comment (Sriparno08) [inline content/pytorch/concepts/tensors/terms/linspace/linspace.md:31]: Let's change this line a bit:

```suggestion
This function is particularly useful when there is a need to create a tensor of equally spaced points for plotting graphs or for performing other numerical computations.
```

File: content/pytorch/concepts/tensors/terms/linspace/linspace.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+
+## Example
+
+Let's see an example of how to use the `.linspace()` function in PyTorch:
```
Comment (Sriparno08) [inline content/pytorch/concepts/tensors/terms/linspace/linspace.md:35]: Let's change this line a bit:

```suggestion
The following example shows how to use the `.linspace()` function in PyTorch:
```

File: content/pytorch/concepts/tensors/terms/linspace/linspace.md [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+
+```py
+import torch
+
+# Create a tensor of 10 equally spaced points between 0 and 1
+x = torch.linspace(0, 1, 10)
+print(x)
+```
```
Comment (Sriparno08) [inline content/pytorch/concepts/tensors/terms/linspace/linspace.md:43]: Let's change the example a bit for uniqueness:

````suggestion
```py
import torch

# Create a tensor of 5 equally spaced points between 0 and 1
x = torch.linspace(0, 1, 5)

print(x)
```
````

File: content/pytorch/concepts/tensors/terms/linspace/linspace.md [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+
+The code above generates the following output:
+
+```shell
+tensor([0.0000, 0.1111, 0.2222, 0.3333, 0.4444, 0.5556, 0.6667, 0.7778, 0.8889, 1.0000])
+```
+
+In this example, we created a tensor `x` containing 10 equally spaced points between 0 and 1 using the `.linspace()` function. The tensor `x` contains the values `[0.0000, 0.1111, 0.2222, 0.3333, 0.4444, 0.5556, 0.6667, 0.7778, 0.8889, 1.0000]`.
+
+The `.linspace()` function is a convenient way to create a tensor of equally spaced points for various numerical computations in PyTorch.
```
Comment (Sriparno08) [inline content/pytorch/concepts/tensors/terms/linspace/linspace.md:53]: Modifications:

- Let's insert the output of the modified code
- Let's remove the second last paragraph as it's not needed
- Let's remove the last line as it's already mentioned before

````suggestion
```shell
tensor([0.0000, 0.2500, 0.5000, 0.7500, 1.0000])
```
````

**Review by Sriparno08 — CHANGES_REQUESTED** [review]
Hey @arisdelacruz, thanks for contributing to Codecademy Docs!

I've listed all the modifications that you need to make in your entry below:

**Review by Sriparno08 — APPROVED** [review]
Good job, @arisdelacruz!

This entry looks good for a second round of review.

**Review by mamtawardhani — APPROVED** [review]
Thank you for contributing @arisdelacruz 😄 

The entry is ready to be merged! 🚀
